spec/base_service_spec.rb

```diff
@@ -126,13 +140,16 @@
   end
 
   context 'have method to check if options value is enabled/disabled' do
-    subject { NiftyServices::BaseService.new(send_push_notification: true, create_users: false) }
+    subject do
+      NiftyServices::BaseService.new(send_push_notification: true,
+                                     create_users: false)
+    end
 
-    it { expect(subject.option_enabled?(:send_push_notification)).to be_truthy }
-    it { expect(subject.option_enabled?(:create_users)).to be_falsey }
+    it { expect(subject.option_enabled?(:send_push_notification)).to be true }
```

Comment:
@viniciuszago I don't see why changes made by @lucasgomide  was undone here. (I'm referring to `be_truthy`, `be_falsey` changes.